src/ext/resources/gtfsgraphqlapi/schema.graphqls

```diff
@@ -3394,6 +3394,65 @@ enum AbsoluteDirection {
   NORTHWEST
 }
 
+"""Occupancy status of a vehicle."""
+enum OccupancyStatus {
+  """Default. There is no occupancy-data on this departure."""
+  NO_DATA_AVAILABLE
+
+  """
+  The vehicle is considered empty by most measures, and has few or no passengers onboard, but is
+  still accepting passengers. There isn't a big difference between this and MANY_SEATS_AVAILABLE
+  so it's possible to handle them as the same value, if one wants to limit the number of different
+  values.
+  """
+  EMPTY
+
+  """
+  The vehicle or carriage has a large number of seats available. The amount of free seats out of
+  the total seats available to be considered large enough to fall into this category is
+  determined at the discretion of the producer. There isn't a big difference between this and
+  EMPTY so it's possible to handle them as the same value, if one wants to limit the number of
+  different values.
+  """
+  MANY_SEATS_AVAILABLE
+
+  """
+  The vehicle or carriage has a small number of seats available. The amount of free seats out of
+  the total seats available to be considered small enough to fall into this category is
+  determined at the discretion of the producer.
+  """
+  FEW_SEATS_AVAILABLE
+
+  """The vehicle or carriage can currently accommodate only standing passengers."""
+  STANDING_ROOM_ONLY
+
+  """
+  The vehicle or carriage can currently accommodate only standing passengers and has limited
+  space for them. There isn't a big difference between this and FULL so it's possible to handle
+  them as the same value, if one wants to limit the number of different values.
+  """
+  CRUSHED_STANDING_ROOM_ONLY
+
+  """
+  The vehicle is considered full by most measures, but may still be allowing passengers to
+  board.
+  """
+  FULL
+
+  """
+  The vehicle or carriage is not accepting passengers. The vehicle or carriage usually accepts
+  passengers for boarding.
+  """
```

Comment:
I know that you copied this from the GTFS doc, but I find these two sentences contradictory.